### PR TITLE
Updated copyright date

### PR DESCRIPTION
**Pull request by qgisdocstutorials**
The copyright date was outdated, I have bumped it to 2019.

### Description

Goal:

Ticket: #

- [ ] Backport to LTR documentation is required

### Minimal requirements for merging *(for maintainers)*

- [ ] The description of this PR is coherent with the manual and does not provide wrong information.
- [ ] This PR passes the checks.

### CODE REVIEW

**Comment by qgisdocstutorials** [comment]
I am using this to create a tutorial about how to contribute to QGIS Documentation, please don't merge it. (@SrNetoChan)

File: source/docs/user_manual/preamble/preamble.rst [hunk 1]
```diff
@@ -46,7 +46,7 @@ form, the browser displays and handles both identically.
 The list of the persons who contribute on writing, reviewing and translating
 the following documentation is available at :ref:`doc_contributors`.
 
-Copyright (c) 2004 - 2017 QGIS Development Team
+Copyright (c) 2004 - 2019 QGIS Development Team
```
Comment (SrNetoChan) [inline source/docs/user_manual/preamble/preamble.rst]: While we are working with this, let's remove the space between the two dates

```suggestion
Copyright (c) 2004-2019 QGIS Development Team
```

Comment (SrNetoChan) [reply source/docs/user_manual/preamble/preamble.rst]: Can we also add a period at the end of the sentence, please?

**Comment by qgisdocstutorials** [comment]
@Srnetochan I made all the changes you request. can you please review? thanks

**Comment by havatv** [comment]
All comments / suggestions resolved. Ready to merge now, @SrNetoChan ?

**Comment by SrNetoChan** [comment]
@havatv not sure if you understood, but this PR is being used in a video showing how to contribute to QGIS. Please let's keep it opened, I want to use it for more videos. :blush:

**Comment by havatv** [comment]
I think I understood.  But I guess you won't be using it forever.  Please merge when you are done with the videos.

**Comment by SrNetoChan** [comment]
Yes, I need to get more time for completing the videos.

**Comment by havatv** [comment]
@SrNetoChan, are you done with the videos? If so, should this be merged or closed?

**Comment by SrNetoChan** [comment]
I wasn't able to finish the last video. If we could keep this open, I would appreciate it.

Otherwise, close it, and I will reopen when needed

File: source/docs/user_manual/preamble/preamble.rst [hunk 2]
```diff
@@ -46,7 +46,7 @@ form, the browser displays and handles both identically.
 The list of the persons who contribute on writing, reviewing and translating
 the following documentation is available at :ref:`doc_contributors`.
 
-Copyright (c) 2004 - 2017 QGIS Development Team
+Copyright (c) 2004-2019 QGIS Development Team.
```
Comment (agiudiceandrea) [inline source/docs/user_manual/preamble/preamble.rst:49]:
```suggestion
Copyright (c) 2004-2020 QGIS Development Team.
```

We are at 2020 now

**Comment by ghtmtt** [comment]
@qgisdocstutorials can you commit the suggestion of @agiudiceandrea and rebase the branch to get rid of conflict. After that, @SrNetoChan @havatv can this PR be merged or are still reasons to let it opened? Thanks!

**Comment by agiudiceandrea** [comment]
@ghtmtt in the meantime the copyright notice in the preamble has already been updated with PR https://github.com/qgis/QGIS-Documentation/pull/5886 (for testing docs) and https://github.com/qgis/QGIS-Documentation/pull/5889 (for 3.10).

**Comment by SrNetoChan** [comment]
No, I will just close it. I end up not having time to finish the videos anyway

**Comment by ghtmtt** [comment]
thanks to both @agiudiceandrea and @SrNetoChan

**Pull request closed without merging**